api/spec/requests/spree/api/v2/storefront/cart_spec.rb

```diff
@@ -194,4 +194,33 @@
       it_behaves_like 'emptying the order'
     end
   end
+
+  describe 'cart#set_quantity' do
```

Comment:
@WitekL please also include test with insufficient quantity scenario

Reply:
@damianlegawiec I've tested it on service level here: https://github.com/spark-solutions/spree/blob/6e053002d1afe6025000778704adea0d2b660815/core/spec/services/spree/cart/set_quantity_spec.rb#L26 
Should I also test it in the request spec?